doc/configuring_your_machine.md

```diff
@@ -35,14 +35,27 @@ The Azure Python CLI projects sources are located on GitHub (https://github.com/
 
 
 #### Visual Studio Code (Any platform)
-Experimental steps – still haven’t been able to get virtual environments to work well with VSCode
 
-1. Install VS Code
-2. Install (one of) the python extension(s) (https://marketplace.visualstudio.com/items?itemName=ms-python.python)
-3. Put [breakpoints](https://code.visualstudio.com/docs/editor/debugging#_breakpoints) in your code, then click on the Debug icon in the Activity Bar on the side of Visual Studio Code, and select `Azure CLI Debug` from the drop down menu. After that, press the play button to start debugging.
-4. To add custom arguments to the command that you want to debug, press the gear icon beside the drop down menu and modify the `args` field in `launch.json`
 
-The repo has a `launch.json` file that will launch the version of Python that is first on your path.
+1. Install [VS Code](https://code.visualstudio.com/)
+2. Install (one of) the [python extension(s)](https://marketplace.visualstudio.com/items?itemName=ms-python.python) for VS Code
+3. Create a new [virtual environment](https://docs.python.org/3/library/venv.html)
+    - The location for the environment does not matter, but it's recommended to create it as close to the root folder as possible
+    - If there is an existing virtual environment you wish to use, skip this step 
+4.  Open the folder containing the clone of this repository in VS Code
+5. Set the [Python Interpreter](https://code.visualstudio.com/docs/python/environments#_select-and-activate-an-environment) to use as the python.exe located in your virtual environment
+    - If your virtual environment's Python Interpreter is not shown you may need to add the root folder it's located in to the extension's venv path. This will be under File -> Preferences -> Settings -> Python: Venv Path.
```

Comment:
```suggestion
    - If your virtual environment's Python Interpreter is not shown, you may need to add the root folder it's located in to the extension's venv path. This will be under File -> Preferences -> Settings -> Python: Venv Path.
```

Reply:
Just made the update in the latest commit. Thank you for your feedback on this.